tests/runtime/call

```diff
@@ -227,32 +227,32 @@ TIMEOUT 5
 AFTER ./testprogs/syscall nanosleep  1e8
 
 NAME ustack_stack_mode_env_bpftrace
-PROG k:do_nanosleep { printf("%s", ustack(1)); exit(); }
+PROG u:./testprogs/uprobe_loop:uprobeFunction1 { printf("%s", ustack(1)); exit(); }
 ENV BPFTRACE_STACK_MODE=bpftrace
-EXPECT_REGEX ^\s+[a-zA-Z0-9_]+
+EXPECT uprobeFunction1+0
 TIMEOUT 5
-AFTER ./testprogs/syscall nanosleep  1e8
+AFTER ./testprogs/uprobe_loop
 
 NAME ustack_stack_mode_env_perf
-PROG k:do_nanosleep { printf("%s", ustack(1)); exit(); }
+PROG u:./testprogs/uprobe_loop:uprobeFunction1 { printf("%s", ustack(1)); exit(); }
 ENV BPFTRACE_STACK_MODE=perf
-EXPECT_REGEX ^\s+[0-9a-f]+ [a-zA-Z0-9_]+
+EXPECT_REGEX \d+ uprobeFunction1\+0 \([a-zA-Z0-9_/]+/uprobe_loop\)
 TIMEOUT 5
-AFTER ./testprogs/syscall nanosleep  1e8
+AFTER ./testprogs/uprobe_loop
 
 NAME ustack_stack_mode_env_raw
-PROG k:do_nanosleep { printf("%s", ustack(1)); exit(); }
+PROG u:./testprogs/uprobe_loop:uprobeFunction1 { printf("%s", ustack(1)); exit(); }
 ENV BPFTRACE_STACK_MODE=raw
-EXPECT_REGEX ^\s+[0-9a-f]+$
+EXPECT_REGEX ^\d+$
```

Comment:
Doesn't this regex need to match hexadecimal numbers?

Reply:
Doesn't seem like it:

```
$ sudo BPFTRACE_STACK_MODE=raw ./build/src/bpftrace  -e 'u:./build/tests/testprogs/uprobe_test:uprobeFunction1 { printf("%s", ustack(1)); exit(); }' -c ./build/tests/testprogs/uprobe_test
[sudo] password for dxu:
Attaching 1 probe...

401136

```

Reply:
I think this is just chance!

I get a similar result:
```
Attaching 1 probe...                                                                                                                                                  
                                                                                                                                                                      
401126
```

But `nm` shows that this value is a hex-address, which just happens to not contain a-f characters:
```
$ nm -p build/tests/testprogs/uprobe_test 
000000000040037c r __abi_tag
0000000000401080 t deregister_tm_clones
00000000004010b0 t register_tm_clones
00000000004010f0 t __do_global_dtors_aux
0000000000404031 b completed.0
0000000000403e08 d __do_global_dtors_aux_fini_array_entry
0000000000401120 t frame_dummy
0000000000403e00 d __frame_dummy_init_array_entry
0000000000402118 r __FRAME_END__
0000000000403e10 d _DYNAMIC
0000000000402010 r __GNU_EH_FRAME_HDR
0000000000404000 d _GLOBAL_OFFSET_TABLE_
                 U __libc_start_main@GLIBC_2.34
                 w _ITM_deregisterTMCloneTable
0000000000404020 W data_start
0000000000404031 D _edata
000000000040113b T uprobeFunction2
0000000000401200 T _fini
000000000040402c D GLOBAL_C
0000000000404024 D GLOBAL_A
0000000000404020 D __data_start
                 w __gmon_start__
0000000000402008 R __dso_handle
0000000000402000 R _IO_stdin_used
0000000000404038 B _end
0000000000401070 T _dl_relocate_static_pie
0000000000401040 T _start
0000000000404031 B __bss_start
000000000040114d T main
0000000000401126 T uprobeFunction1
0000000000404038 D __TMC_END__
                 w _ITM_registerTMCloneTable
0000000000404030 D GLOBAL_D
0000000000404028 D GLOBAL_B
0000000000401000 T _init
                 U usleep@GLIBC_2.2.5
```

Reply:
Ha, nice catch!